deploy/lib/server_config.rb

```diff
@@ -536,6 +536,19 @@ def config
   def bootstrap
     raise ExitException.new("Bootstrap requires the target environment's hostname to be defined") unless @hostname.present?
 
+    if @server_version > 7 && @properties["ml.app-type"] == 'rest' && @properties["ml.url-rewriter"] == "/MarkLogic/rest-api/rewriter.xqy"
+      @logger.info "WARN: XQuery REST rewriter has been deprecated since MarkLogic 8"
+      @properties["ml.url-rewriter"] = "/MarkLogic/rest-api/rewriter.xml"
+    
+    elsif @server_version < 8 && @properties["ml.app-type"] == 'rest' && @properties["ml.url-rewriter"] == "/MarkLogic/rest-api/rewriter.xml"
+      @logger.info "WARN: XML REST rewriter not supported on MarkLogic 7 or less"
+      @properties["ml.url-rewriter"] = "/MarkLogic/rest-api/rewriter.xqy"
+    
+    elsif @server_version > 7 && @properties["ml.app-type"] == 'hybrid'
+      @logger.info "WARN: Running hybrid app-type with MarkLogic 8 requires manual patching of Roxy rewriter."
```

Comment:
@grtjn can you change the warning to this?

"WARN: Running the hybrid app-type with MarkLogic 8 is not recommended. Doing so requires manual patching of the Roxy rewriter. You will be unable to access all of the MarkLogic REST features."